app/controllers/api/ingredients_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::IngredientsController < ApplicationController
+  def index
+    ingredients = [
+      "Coca-Cola", 
+      "Lemon", 
+      "Milk", 
+      "Egg white", 
+      "Lime juice", 
+      "Sugar Syrup", 
+      "Whipping cream", 
+      "Orange juice", 
+      "Lemon juice", 
+      "Coffee", 
+      "Lime", 
+      "Lemon", 
+      "Orange", 
+      "Pineapple juice", 
+      "Tonic Water", 
+      "Mint", 
+      "Ginger", 
+      "Ice", 
+      "Ginger ale", 
+      "Passion fruit juice"
+    ]
```

Comment:
use this instead:

```ruby
class Api::IngredientsController < ApplicationController
  AVAILABLE_INGREDIENTS = ["Coca-Cola", "Lemon", "Milk"]

  def index
     render json: { ingredients: Api::IngredientsController.AVAILABLE_INGREDIENTS }
  end
end
```

Reply:
@tochman I tried to change it as you suggested, but then i got and error saying that AVAILABLE_INGREDIENTS was and undefined method, but without Api::Ingredientscontroller before the tests were passing again. I do not know if i did something wrong and tried to find my answer but couldn't really be any wiser from it. 
Please check the changes I made and tell me if i got this wrong :)